Commands/Reformat Document.tmCommand

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>beforeRunningCommand</key>
+	<string>saveActiveFile</string>
+	<key>command</key>
+	<string>#!/bin/bash
+
+[[ -f "$HOME/.rvm/scripts/rvm" ]] &amp;&amp; . "$HOME/.rvm/scripts/rvm"
+
+(
+bundler_dir="${TM_PROJECT_DIRECTORY:-$(dirname "$TM_FILEPATH")}"
+if [[ -f "$bundler_dir/Gemfile" ]]; then
+  cd "$bundler_dir" &amp;&amp; bundle list rubocop
+  if [[ $? == 0 ]]; then bundler_rubocop='bundle exec rubocop'; fi
+fi
+
+if [[ ! `which rubocop` &amp;&amp; -z "$bundler_rubocop" ]]; then
+  "$DIALOG" tooltip --text "This command requires RuboCop.
+Please install RuboCop via “gem install rubocop”."
+  exit
+fi
+
+aha_found=`which aha`
+if [[ $aha_found ]]; then
+  color='n';
+  format='--html'
+else
+  format='--text'
+fi
+
+rubocop="${TM_RUBOCOP:-"${bundler_rubocop:-rubocop}"}"
+output=`$rubocop -a$color "$TM_FILEPATH"`
+if [[ $aha_found ]]; then output=`echo "$output" | aha`; fi
+
+"$DIALOG" tooltip "$format" "$output"
+) &amp;&gt;/dev/null &amp;</string>
+	<key>input</key>
+	<string>none</string>
+	<key>inputFormat</key>
+	<string>text</string>
+	<key>keyEquivalent</key>
+	<string>^H</string>
+	<key>name</key>
+	<string>Reformat Document</string>
+	<key>outputCaret</key>
+	<string>interpolateByChar</string>
+	<key>outputFormat</key>
+	<string>text</string>
+	<key>outputLocation</key>
+	<string>discard</string>
+	<key>requiredCommands</key>
+	<array>
+		<dict>
+			<key>command</key>
+			<string>rubocop</string>
+			<key>locations</key>
+			<array>
+				<string>/usr/local/bin/rubocop</string>
+				<string>$HOME/.rbenv/shims/rubocop</string>
+				<string>$HOME/.rvm/scripts/rvm</string>
```

Comment:
> It depends on how Ruby is installed. If it's installed using RVM then the path of Rubocop will be something like:
> `~/.rvm/gems/ruby-<version>/bin/rubocop`, where `<version>` is the version of Ruby used when Rubocop was installed.

In that case the location `$HOME/.rvm/scripts/rvm` should exist (line [62][]). The bash code will then source  `$HOME/.rvm/scripts/rvm` (line [10][]). After that `rubocop` is available via `PATH`. If it is not, then `rubocop` was not installed via `rvm` and the script will complain (line [19][]).
[10]: https://github.com/textmate/ruby.tmbundle/pull/99/files#diff-f65299d257213d73ed56dd8823c1335cR10
[19]: https://github.com/textmate/ruby.tmbundle/pull/99/files#diff-f65299d257213d73ed56dd8823c1335cR19
[62]: https://github.com/textmate/ruby.tmbundle/pull/99#discussion-diff-71761612R62